### ISSUE TITLE
revamp parsnip model list for censored

### ISSUE THREAD

**Issue by topepo** [tidymodels/tidymodels.org-legacy#232]
The model mode is broken into "regression", "classification" and essentially "both". We should probably change the nature of this column to accommodate the censored package.

**Comment by juliasilge** [comment]
Do you mean the table at https://www.tidymodels.org/find/parsnip/?

**Comment by hfrick** [comment]
@juliasilge yes, that is what we talked about. I said it'd be handy to be able to find a list of all available model/engine combinations for say classification. Currently you could get that by filtering for "classification" and "both" but when adding the censored regression mode that gets fairly messy.
